main.tf

```diff
@@ -65,8 +65,27 @@ resource "google_service_account_iam_binding" "workload_identity_binding" {
   members            = [for each in var.host_aws_instance_roles : format("principalSet://iam.googleapis.com/projects/%s/locations/global/workloadIdentityPools/%s/attribute.aws_role/arn:aws:sts::%s:assumed-role/%s", data.google_project.my_host_project.number, google_iam_workload_identity_pool.create_wip.workload_identity_pool_id, var.host_aws_account_id, each)]
 }
 
+resource "google_project_iam_custom_role" "role_for_uptycs" {
+  role_id     = "roleForUptycs"
```

Comment:
can we create role name based on integration name?
cc: @gbonthu

Reply:
`"${var.integration_name}-role"`
Is this good or do you think it should just be the integration name?